### PR TITLE
Minor Enhancements for Storage Dashboards

### PR DESCRIPTION
**Pull request by afreen23**
- As per UXD suggestions, made these minor enhancements:

   * Fixed width of bar in breakdown card
   * Spanning of lines to full width in utilization card
   * Changing tab names
   * Linking of Nodes

- Changed ceph plugin gating from cephcluster model to storagecluster model

Screenshot from 2020-01-16 14-34-34
Screenshot from 2020-01-16 13-36-53
Screenshot from 2020-01-16 14-33-50

/assign @yuvalgalanti

### CODE REVIEW

**Comment by cloudbehl** [comment]
@yuvalgalanti Can you check the breakdown card.

other things looks good to me.

File: frontend/packages/ceph-storage-plugin/src/plugin.ts [hunk 1]
```diff
@@ -45,16 +45,16 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'FeatureFlag/Model',
     properties: {
-      model: models.CephClusterModel,
-      flag: CEPH_FLAG,
+      model: models.OCSServiceModel,
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/plugin.ts]: just the model as the flag is generic for the ceph extension we don't need to give name corresponding to model we are using.

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/plugin.ts]: Lets not fix that is not broken

**Comment by spadgett** [comment]
/hold
the merge queue is blocked

**Pull request merged**